client/.vscode/settings.json

```diff
@@ -12,5 +12,7 @@
     "typescript.tsc.autoDetect": "off",
     "ocaml.sandbox": {
         "kind": "global"
-    }
+    },
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.formatOnSave": true,
```

Comment:
Question beyond this PR, should these settings be tracked?

Reply:
I actually don't know. My feeling is probably not ?